manifest.json

```diff
@@ -3,7 +3,7 @@
 
     "name": "Reddit Nav",
     "description": "This extension allows you to easily scroll through comment threads on Reddit",
-    "version": "1.0.3",
+    "version": "1.0.4",
```

Comment:
Could you leave the version number at `1.0.3` and we'll bump the version when we do a release, thanks.